packages/frontend-api/src/Model/Resolver/Products/ProductsResolver.php

```diff
@@ -38,7 +38,7 @@ public function __construct(
      */
     public function resolve(): array
     {
-        return $this->productOnCurrentDomainFacade->getAllListableProducts();
+        return $this->productOnCurrentDomainFacade->getAllOfferedProducts();
```

Comment:
According to previous changes I'm just curious if we want to use `ProductOnCurrentDomainFacade` only. It will make sense to me stick to the interface `ProductOnCurrentDomainFacadeInterface` and have the same functionality implementent even in `ProductOnCurrentDomainElasticFacade`. Then anyone could just switch the implementation in `services.yml`. But this is not the scope of this PR maybe we should create an issue or new PR for this. What do you think?